### PR TITLE
create the "remove old directory option"

### PR DESCRIPTION
**Pull request by nathandunn**

### CODE REVIEW

File: tools/apollo/create_or_update_organism.py [hunk 1]
```diff
@@ -39,6 +41,9 @@
 
     if org:
         has_perms = False
+        old_directory = org.directory
+
+
```
Comment (erasche) [inline tools/apollo/create_or_update_organism.py]: can you remove the extra spaces please?

Comment (nathandunn) [inline tools/apollo/create_or_update_organism.py]: done

**Comment by nathandunn** [comment]
@erasche  Ready to merge.

**Comment by erasche** [comment]
Sorry I'd forgotten about this / missed it.

**Pull request merged**